base/xx-cc

```diff
@@ -2,7 +2,7 @@
 
 set -e
 
-mirrors="https://github.com/tonistiigi/xx/releases/download/prebuilt%2Fld-1"
+mirrors="https://github.com/crazy-max/xx/releases/download/prebuilt%2Fld-2.38-34"
```

Comment:
Could you copy this to this repo. You can leave the release as draft like the old one